app/build.gradle

```diff
@@ -168,8 +168,7 @@ dependencies {
     androidTestCompile 'com.android.support.test:runner:0.5'
     apt 'com.jakewharton:butterknife-compiler:8.4.0'
     apt 'org.projectlombok:lombok:1.16.10'
-    onlineCompile 'com.squareup.okhttp3:okhttp:3.4.1'
-    onlineCompile 'com.squareup.okio:okio:1.10.0'
```

Comment:
okio is a [transitive dependency](https://github.com/square/okhttp/blob/master/okhttp/pom.xml#L17) of okhttp so specifying it here doesn't achieve much